overrides.nix

```diff
@@ -89,11 +93,14 @@ in {
     sha256 = "1isa8p9dnahkljwj0kz10119dwiycf11jvzdc934lnjv1spxkc9k";
   });
 
-  singleton-bool = dontCheck (callHackageDirect {
-    pkg = "singleton-bool";
-    ver = "0.1.5";
-    sha256 = "1kjn5wgwgxdw2xk32d645v3ss2a70v3bzrihjdr2wbj2l4ydcah1";
-  });
+  #https://github.com/reflex-frp/reflex-platform/issues/549
+  singleton-bool = overrideCabal
+    (self.callCabal2nix "singleton-bool" (hackGet ./deps/singleton-bool) {})
```

Comment:
If we're using hackGet here, should we use it for `aeson` and `servantSrc` as well?

Reply:
Oh I don't have a preference, I had it like this while doing a lot of testing locally, but I've pushed it now so this could be a normal `fetchFromGitHub`